packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -7772,6 +7772,114 @@ QUnit.module('label integration', () => {
 });
 
 QUnit.module('accessibility', () => {
+    QUnit.test('Custom aria-label attribute should be added to input if inputAttr property is used', function(assert) {
+        const productsData = [
+            'HD Video Player',
+            'SuperHD Video Player',
+            'SuperPlasma 50',
+        ];
+        const productLabel = { 'aria-label': 'Product' };
+        const $tagBox = $('#tagBox').dxTagBox({
+            items: productsData,
+            inputAttr: productLabel,
+            value: [...productsData],
```

Comment:
Do we really need to create another array? I think it should work correctly if you just use the value: productsData. Please check this for such cases.

Reply:
[b2bc2ea](https://github.com/DevExpress/DevExtreme/pull/27157/commits/b2bc2ea9b8b514617e2f15d5f07373ddafcd8195)